tests/test_abstract_storage.py

```diff
@@ -3,31 +3,44 @@
 import time
 
 from aiohttp import web
+from aiohttp.web_middlewares import _Handler
+from aiohttp.test_utils import TestClient
+
+from typing import Any, Dict
+
 from aiohttp_session import get_session, SimpleCookieStorage
 from aiohttp_session import setup as setup_middleware
 
+from typedefs import _TAiohttpClient
+
 
-def make_cookie(client, data):
+def make_cookie(client: TestClient, data: Dict[Any, Any]) -> None:
     session_data = {
         'session': data,
         'created': int(time.time())
     }
 
     value = json.dumps(session_data)
-    client.session.cookie_jar.update_cookies({'AIOHTTP_SESSION': value})
+    # Ignoring type until aiohttp#4252 is released
+    client.session.cookie_jar.update_cookies(
+        {'AIOHTTP_SESSION': value}  # type: ignore
+    )
 
 
-def create_app(handler):
+def create_app(handler: _Handler) -> web.Application:
     app = web.Application()
     setup_middleware(app, SimpleCookieStorage(max_age=10))
     app.router.add_route('GET', '/', handler)
     return app
 
 
-async def test_max_age_also_returns_expires(aiohttp_client):
+async def test_max_age_also_returns_expires(
+    aiohttp_client: _TAiohttpClient
+) -> None:
 
-    async def handler(request):
-        time.time.return_value = 0.0
+    async def handler(request: web.Request) -> web.StreamResponse:
+        # Ignoring type since time.time is mocked in this context
+        time.time.return_value = 0.0  # type: ignore
```

Comment:
Maybe a task for another PR, but ideally we shouldn't be referencing mock objects in this way, we should have a direct reference to the mock which we can safely modify without risk of affecting it's state.

Reply:
I would keep it for another PR too, I've tried to keep code changes as minimal as possible in this one.